pages/index.js

```diff
@@ -1,13 +1,20 @@
+import React from "react";
 import Link from "next/link";
+import Image from "next/image";
 import Footer from "../components/Footer/index.js";
 
 export default function HomePage() {
   return (
     <main>
+      <header>
+        <Link href="/">
+          <Image src="/images/logo1.png" alt="Logo" width={200} height={100} />
+        </Link>
+      </header>
```

Comment:
you created it as an header here, but on the other pages you just added the picture, but not the header? 

you could change it into an header-component. there you could also make it fixed so it stays on top :)

then you could just import it into your pages